docs/source/configuration/metrics.mdx

```diff
@@ -102,7 +102,8 @@ The coprocessor operations metric has the following attributes:
 #### Performance
```

Comment:
There metrics are in prom format. You'll have to somehow capture this as in datadog and anything otlp based they'll be displayed with dot rather than underscore.